frontend/src/teacher/classes.js

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from 'react';
+import ReactDOM from "react-dom";
+import BurgerMenu from '../components/Burger';
+import { Button, Form } from 'semantic-ui-react';
+import '../styles/teacherClasses.css';
+import { classCreateRequestor, getClassRequestor } from '../requests/requestBuilder'
+
+const fetchClasses = async () => {
+  let classes = getClassRequestor(localStorage.getItem('userid'), localStorage.getItem('usertype'))
+    .then(res => {
+      return res.data;
```

Comment:
This promise wont really work. I'll get on your branch later and show you how to fetch data.